mesonbuild/linkers.py

```diff
@@ -909,3 +909,33 @@ def __init__(self, for_machine: mesonlib.MachineChoice,
 
     def get_allow_undefined_args(self) -> typing.List[str]:
         return []
+
+class CudaLinker(DynamicLinker):
+    """Cuda linker (nvlink)"""
+    @staticmethod
+    def parse_version():
+        version_cmd = ['nvlink', '--version']
+        try:
+            p, out, err = mesonlib.Popen_safe(version_cmd)
+        except OSError as e:
```

Comment:
__[Flake8]__ [F841] local variable 'e' is assigned to but never used ([link](https://sider.review/gh/repos/19784232/pulls/5874#issue-4504460))

<sub>posted by Sider</sub>

Reply:
fixed